frame/staking/src/lib.rs

```diff
@@ -732,6 +731,91 @@ impl<AccountId, Balance: Default + HasCompact> Default for Exposure<AccountId, B
 	}
 }
 
+impl<AccountId: Clone, Balance: HasCompact + AtLeast32BitUnsigned + Copy>
+	Exposure<AccountId, Balance>
+{
+	/// Splits an `Exposure` into `ExposureOverview` and multiple chunks of `IndividualExposure`
+	/// with each chunk having maximum of `page_size` elements.
+	fn as_pages(
+		&self,
```

Comment:
```suggestion
		self,
```

Seems like we want to consume self and return the chunked version, doing so will save memory use as well ✅